docs/en/manuals/lua.md

```diff
@@ -600,13 +603,13 @@ Use the value returned from `socket.gettime()` (seconds since system epoch) to b
 
 ## Memory and garbage collection
 
-Lua's garbage collection runs automatically in the background by default and reclaims memory that the Lua runtime has allocated. Collecting lots of garbage can be a time consuming task so it is good to keep down the number of objects that needs to be garbage collected:
+Lua's garbage collection runs automatically in the background by default and reclaims memory that the Lua runtime has allocated. Collecting lots of garbage can be a time-consuming task so it is good to keep down the number of objects that need to be garbage collected:
 
 * Local variables are in themselves free and will not generate garbage. (i.e. `local v = 42`)
 * Each _new unique_ string creates a new object. Writing `local s = "some_string"` will create a new object and assign `s` to it. The local `s` itself will not generate garbage, but the string object will. Using the same string multiple times adds no additional memory cost.
 * Each time a table constructor is executed (`{ ... }`) a new table is created.
 * Executing a _function statement_ creates a closure object. (i.e. executing the statement `function () ... end`, not calling a defined function)
-* Vararg functions (`function(v, ...) end`) create a table for the ellipsis each time the function is _called_ (in Lua prior to version 5.2, or if not using LuaJIT).
+* Vararg functions (`function(v, ...) end`) create a table for the ellipsis each time the function is _called_ (in Lua before version 5.2, or if not using LuaJIT).
```

Comment:
I think the previous text was fine.
In general, I think rewordings are a bit unnecessary, unless the meaning was wrong.
Pls bear in mind that we need to review all changes.

Reply:
Well, the grammar was wrong in many places and the info about 5.3 was definitely missing.

Is anything going worse, with merging this commit?

Reply:
No, we appreciate the improvements! Thank you.

The point I think @JCash is trying to make with "prior to" vs "before" is that both are grammatically correct, but "prior to" is more "formal". In general I think it is a good idea to simplify the text and use "simpler" works (eg "before") when it helps the reader understand the text.

But let's also for a moment consider what would happen if we sent all of the manuals to a service such as Grammarly. There would probably be several hundred (thousands?) of changes that we would have to review. Can we cut down the amount of work by 5 or 10% by not changing synonyms?

Reply:
Half of the people reading this text, do have not read "prior to" in their entire life.

A lot of people use English as their second language, and making the documentation more accessible to them, seems sensible to me. 

I assume someone with a background from a English speaking country, does have a different perspective on that.

Ultimately, I saw no change, that would make the text less readable, so I decided to trust them.

I volunteer to review changes made to the documentation.

And when we use software like that to begin with, such changes would not happen at all, cutting down the review costs to 0.

Reply:
> And when we use software like that to begin with, such changes would not happen at all, cutting down the review costs to 0

I would not blindly trust an AI grammar service to correctly update a technical manual. There will be a review cost.

Reply:
I would not blindly trust it either. Grammarly works for me since 7 years, also before it became 'AI' driven. 

Obviously, we (I) review the output first.

In fact, I have to accept and each single word manually to begin with.